### PR TITLE
Updated portuguese translations

### PR DESCRIPTION
**Pull request by bfavaretto**
Existing translations revised, missing translations added. Tried to be as neutral as possible concerning eurupean vs brazilian portuguese (disclaimer: I am brazilian).

Also added a new block at the end of the translation file for missing glossary config menu item.

### CODE REVIEW

File: common/lang/pt-utf-8.inc.php [hunk 1]
```diff
@@ -415,100 +415,104 @@
...
+define('LABEL_FORM_NULLnotesTermReport','termos SEM notas');
+define('MSG_FORM_NULLnotesTermReport','termos sem notas do tipo');
+define('LABELnoNotes','termos sem nenhuma nota');
+define('LABEL_termsXdeepLevel','termos por nível hierárquico');
+define('LABEL_deepLevel','nível');
+define('LABEL_cantTerms','# de termos');
+define('LINK_publicKnownVocabularies','<a href="http://www.vocabularyserver.com/vocabularies/" title="Lista de vocabularios controlados conhecidos" target="_blank">Lista de vocabulários controlados conhecidos</a>');
+define('LABEL_showNewsTerm','ver alterações recentes');
+define('LABEL_newsTerm','alterações recentes');
+define('MSG_contactAdmin','contace o administador');
```
Comment (horus68) [inline common/lang/pt-utf-8.inc.php]: Misspell error: `contace`

**Comment by horus68** [comment]
There is another PR waiting to be merged https://github.com/tematres/TemaTres-Vocabulary-Server/pull/19

Note: I had already requested a new language to split portuguese into portuguese Portugal (european) and Brazil.
I am from Portugal and can translate the portuguese (Portugal).

File: common/lang/pt-utf-8.inc.php [hunk 2]
```diff
@@ -415,100 +415,104 @@
...
+define('MSG_termsSuccessTask','termos afetados pela operação');
+define('LABEL_TTTerms','termos de topo');
+define('MSG__GLOSSincludeAltLabel','incluir termos alternativos');
+define('MSG__GLOSSdocumentationJSON','A fonte JSON do glossário pode ser integrada com qualquer conteúdo HTML por meio da biblioteca <a href="https://github.com/PebbleRoad/glossarizer" target="_blank" title="Glossarizer">Glossarizer</a>');
+define('LABEL_configGlossary','exportar fonte de dados para glossário');
+define('MSG_includeNotes','utilizar notas do tipo:');
+define('LABEL_SHOW_RANDOM_TERM','exibir um termo aleatório na página inicial. É necessário escolher um tipo de nota.');
+define('LABEL_opt_show_rando_term','exibir termos com nota tipo:');
+define('MSG_helpNoteEditor','Associe termos utilizando colchetes duplos. Ex: Só o [[amor]] salvará o mundo');
+define('LABEL_GLOSS_NOTES','Selecionar tipo de nota para glosar termos marcados com colchetes dobles: [[glossário]]');
```
Comment (horus68) [inline common/lang/pt-utf-8.inc.php]: not fully translated: colchetes dobles

**Comment by tematres** [comment]
Gracias!!!! :)!

**Comment by horus68** [comment]
@tematres what about my request to have 2 different Portuguese language files, one for pt (Portugal) and pt-BR (Brasil)?

**Comment by bfavaretto** [comment]
Hey @horus68 

I was taking a look at the code that generates the language select box, it looks simple to add a new one. Maybe we could do that ourselves and start a now PR?

**Comment by horus68** [comment]
I'm not a skilled coder but count me in to handle a pt (Portugal) file.

**Comment by tematres** [comment]
now there are separated file to pt-PT and to pt-BR :)

**Pull request merged**